source/customization/menus.rst

```diff
@@ -1,5 +1,367 @@
-=====
-Menus
-=====
+=======
+ Menus
+=======
 
-No documentation is available about this topic.
+.. seealso::
+
+   :doc:`Reference for menus </reference/menus>`
+      Reference for menu creation.
+
+Sublime Text provides several menus

```

Comment:
There actually is exactly one menu that can not be customized: Open files in the sidebar.